### PR TITLE
Feature/hands scaling

### PR DESCRIPTION
**Pull request by YasserNezzari**
Scaling the poly hands and the avatar hands

### CODE REVIEW

**Comment by YasserNezzari** [comment]
Tested with smaller hands (cut from white paper) and bigger hands wearing gloves and both poly hands and avatar hands were scaling up and down accordingly, when you click on calibrate button in the hand menu.  Docs here how to use:
https://gitlab.ultrahaptics.com/marcom/ultraleap-api-docs/-/commit/a3ba3b1cb198db6d80731922af5ff82bc3bfdb31

**Comment by Greg-UL** [comment]
Interesting I thought the scaling should be present prior to the calibration button being pressed for the avatar, at least the poly hands definitely should be.
Might be good to get this packaged up and tested on android too before pushing this further.

**Comment by YasserNezzari** [comment]
@Greg-UL  it can be done on tick, but that would be expensive, the reason I done on button press, is for the user to hold their hands in front of the tracking camera, and that would give more precise scaling without affect the performance. but if you want it done without button press no problem

**Comment by Greg-UL** [comment]
I think the poly models should be scaling without any need for the calibration button to be pressed. I don't mind if the avatar ones scale after calibrating

**Comment by YasserNezzari** [comment]
@Greg-UL I just committed code to auto scale the poly hands, without the need for button press https://github.com/ultraleap/UnrealPlugin/pull/115/commits/28bfefafda89c0bbfaa1c33c57f9b034bde4a960

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -205,15 +238,37 @@ build:build-project:
 
     # zip it
     - Compress-Archive -Path "$BUILD_MARKET_PLACE_PATH_UE5" -DestinationPath "$BUILD_MARKET_PLACE_PATH_UE5.zip"
+
```
Comment (MaxPalmer-UH) [inline .gitlab-ci.yml:241]: I wonder if we should add a ticket to make some of this duplicated script code into a local method that can be called twice with the correct UE5/UE5.1 build parameters? I think that would be a good tool sharpening ticket to have for our next sprint.

File: Source/UltraleapTrackingCore/UltraleapTracking.Build.cs [hunk 2]
```diff
@@ -116,7 +116,8 @@ public UltraleapTracking(ReadOnlyTargetRules Target) : base(Target)
 					"LiveLinkInterface",
 					"LiveLinkMessageBusFramework",
 					"BodyState",
-					"PhysicsCore"
+					"PhysicsCore",
+                    "ApplicationCore"
```
Comment (MaxPalmer-UH) [inline Source/UltraleapTrackingCore/UltraleapTracking.Build.cs:120]: Might need to run some form of code layout/formatting pass on these.

File: Source/UltraleapTrackingCore/Private/LeapComponent.cpp [hunk 3]
```diff
@@ -401,4 +401,36 @@ bool ULeapComponent::GetLeapOptions(FLeapOptions& Options)
 		}
 	}
 	return false;
+}
+
+void ULeapComponent::GetHandSize(float& OutHandSize)
```
Comment (MaxPalmer-UH) [inline Source/UltraleapTrackingCore/Private/LeapComponent.cpp:406]: Might be worth commenting this method that we measure the middle finger length + palm position to start of finger as an indication of the hand size.

**Review by MaxPalmer-UH — APPROVED** [review]
Reviewed the non binary code (C++ / YML). Yasser walked me through the blueprint / model changes in editor. Also checked the test scene which works..

**Pull request merged**